licenses/LICENSE.junit-pioneer.epl20.txt

```diff
@@ -0,0 +1,277 @@
+Eclipse Public License - v 2.0
```

Comment:
Thanks for not forgetting to add the license file. Could you also update `NOTICE.txt`?

Reply:
Yeah, exactly. Thanks 🙇

Reply:
It has to be sorted alphabetically by the way.

Reply:
Oh I did't notice 😅. I'll correct that, Thanks.